doc/radosgw/keystone.rst

```diff
@@ -19,7 +19,6 @@ The following configuration options are available for Keystone integration::
 	rgw keystone token cache size = {number of tokens to cache}
 	rgw keystone revocation interval = {number of seconds before checking revoked tickets}
 	rgw keystone implicit tenants = {true for private tenant for each new user}
-	rgw s3 auth use keystone = true
```

Comment:
Something about `rgw_s3_auth_use_keystone` would be nice in `keystone.rst`.  Maybe a link or mention to `s3/authentication.rst`?

Reply:
It's right there in the same patch... paragraph under **Keystone integration with the S3 API**